config.json

```diff
@@ -1,44 +1,44 @@
 {
-    "hostname": "default",
-    "envtype": "TST",
-    "location": "Germany",
-    "serverurl": "http://localhost:9080/hosts",
-    "serverusr": "user",
-    "serverpsw": "password",
-    "frequency": 24,
-    "forcestats": true,
-    "enableservervalidation": false,
-    "forcepwshversion": "0",
-    "oratab": "/etc/oratab",
-    "AWR": 30,
-    "parallelizeRequests": true,
-    "verbose": false,
-    "features": {
-        "databases": {
-            "enabled": true
+    "Hostname": "default",
+    "Environment": "TST",
+    "Location": "Germany",
+    "DataserviceURL": "http://dev.ercole.io/data/hosts",
```

Comment:
DataserviceURL should be without `/hosts`

Reply:
And maybe DataService?

Reply:
...no...